src/main/scala/rocket/ICache.scala

```diff
@@ -220,6 +221,8 @@ class ICacheModule(outer: ICache) extends LazyModuleImp(outer)
 
     ccover(tl_out.d.bits.corrupt, "D_CORRUPT", "I$ D-channel corrupt")
   }
+  io.errors.bus.valid := tl_out.d.fire() && (tl_out.d.bits.denied || tl_out.d.bits.corrupt)
+  io.errors.bus.bits  := refill_paddr
```

Comment:
I think we should do `refill_paddr >> blockOffBits << blockOffBits here` - in addition to shaving off six flops, it keeps things consistent with the D$, which doesn't report the block offset.

Reply:
okay will do

Reply:
@aswaterman  I know it's long time ago, but I felt confused why you suggested refill_paddr >> blockOffBits << blockOffBits instead of refill_paddr,  I thought these two are exactly the same thing. What do you mean by _in addition to shaving off six flops_ Thanks

Reply:
We only report the address of the cache line that had an error, rather than trying to be more precise.

Reply:
> We only report the address of the cache line that had an error, rather than trying to be more precise.

Got it, Thanks!